books/RayTracingTheNextWeek.html

```diff
@@ -2870,7 +3045,7 @@
             constant_medium(shared_ptr<hittable> b, double d, color c)
                 : boundary(b),
                   neg_inv_density(-1/d),
-                  phase_function(make_shared<isotropic>(color(c)))
+                  phase_function(make_shared<isotropic>(make_shared<solid_color>(c)))
```

Comment:
Ooh, missed one. Like other materials, we should have an isotropic constructor that takes a color value. I'll create a separate issue for this: see #644.